### ISSUE TITLE
Génération du pdf en erreur

### ISSUE THREAD

**Issue by RLesur** [InseeFrLab/utilitr-template#19]
Les pipelines qui construisent le pdf avec pagedown sont en erreur, par ex https://github.com/InseeFrLab/utilitR/runs/2640274030

Cela provient du fait que le HTML généré est tronqué (il manque la fin du fichier).

Cela semble dû à un bug dans le package **rmarkdown** lorsqu'on utilise le paramètre `copy_resources = TRUE`. En effet, si l'on met `copy_resources = FALSE` ici, la génération du pdf fonctionne : 

https://github.com/InseeFrLab/utilitr-template/blob/5dfc7ddb44e207e5a783998eec9db2dbd7ae7d11/R/pdf_document.R#L31

Changer `copy_resources` à `FALSE` nécessite d'utiliser des HTML dependencies pour gérer les CSS. 
Deux exemples pour le faire mais il y en a bien d'autres : 
- **pagedown** https://github.com/rstudio/pagedown/blob/f9779e38a4b9652b111fc763c414ea61998cb208/R/paged.R#L125-L132
- **gouvdown** https://github.com/spyrales/gouvdown/blob/653f3c46cba9da09e2c648ccd1a43752eeda22e0/R/html_gouv.R#L93-L107

**Comment by oliviermeslin** [comment]
Merci Romain. J'ai regardé cela hier, sans trouver encore de solution:

- Je confirme que c'est bien `copy_resources = TRUE` qui aboutit à un html tronqué;
- J'ai utilisé les modèles proposés par @RLesur pour construire les dépendances html;
- Je bute sur un point: `bookdown::render_book` ne semble pas avoir d'argument `extra_dependencies`, donc je me demande si on ne va pas être contraints de modifier la fonction `utilitr::pdf_document` pour qu'elle utilise `bookdown::html_document2` (comme [ici](https://github.com/rstudio/pagedown/blob/f9779e38a4b9652b111fc763c414ea61998cb208/R/paged.R#L161)).

**Comment by RLesur** [comment]
Non, non, pas besoin d'utiliser `bookdown::html_document2()`. Il faut faire la différence entre la notion d'output format ([`rmarkdown::output_format()`](https://rmarkdown.rstudio.com/docs/reference/output_format.html)) et la fonction de génération (`rmarkdown::render()` ou `bookdown::render_book`).

Ici le problème est dans l'output format (l'argument `extra_dependencies` est un argument de `rmarkdown::html_document_base()`) pas dans dans la fonction de génération.
